### PR TITLE
UX updates for VWB

### PR DESCRIPTION
**Pull request by dexamundsen**
Changes from past demos, for product demo tomorrow

- sign in page: change text to “Data Explorer”
- Change return app name in VWB export to data explorer
- Other changes in export page based on UX review

image

### CODE REVIEW

**Review by marikomedlock — APPROVED** [review]
You'll need to update the internal deployment repo(s) with the new env var names with this change.

File: service/src/main/java/bio/terra/tanagra/service/export/impl/IpynbFileDownload.java [hunk 1]
```diff
@@ -32,12 +31,12 @@ public Type getType() {
...
   @Override
   public String getDefaultDisplayName() {
-    return "Download ipynb file with embedded SQL";
+    return "Download Jupyter Notebook";
   }
 
   @Override
   public String getDescription() {
-    return "Signed URL to an ipynb notebook file that includes SQL to get the list of primary entity instances.";
+    return "Get an IPYNB file for download. SQL comes preinstalled.";
```
Comment (marikomedlock) [inline service/src/main/java/bio/terra/tanagra/service/export/impl/IpynbFileDownload.java:39]: What does it mean to have SQL preinstalled?

**Review by tjennison-work — APPROVED** [review]

File: service/src/main/java/bio/terra/tanagra/service/export/impl/IpynbFileDownload.java (hunk 1 above)
Comment (Matthew-Antchak) [reply service/src/main/java/bio/terra/tanagra/service/export/impl/IpynbFileDownload.java:39]: > What does it mean to have SQL preinstalled?

My assumption is that it includes a SQL kernel and dependencies

Comment (marikomedlock) [reply service/src/main/java/bio/terra/tanagra/service/export/impl/IpynbFileDownload.java:39]: Do you know if this is true of the existing export implementation?

The generated ipynb file contains the SQL query and the idea is to make it easy to run that query against BigQuery. The template file we use is [here](https://github.com/DataBiosphere/tanagra/blob/main/service/src/main/resources/export/notebook_template.ipynb).

Comment (dexamundsen) [reply service/src/main/java/bio/terra/tanagra/service/export/impl/IpynbFileDownload.java:39]: @Matthew-Antchak how about 'Get an IPYNB file for download`. From the template - it uses pandas, not SQL kernel.

Comment (Matthew-Antchak) [reply service/src/main/java/bio/terra/tanagra/service/export/impl/IpynbFileDownload.java:39]: Yes, in that case let's just omit the second sentence about SQL @dexamundsen

**Pull request merged**